src/components/common/listOfArticles/index.js

```diff
@@ -23,6 +23,12 @@ export default function ListOfArticles({ articles, limitArticlesNumber }) {
           slug={article && article.slug}
           rating={rating}
         />
+        {article.reportReason && (
+          <div className="alert alert-danger" role="alert">
+  Reported due to this reason:
+            {article.reportReason }
+          </div>
+        )}
```

Comment:
Put this inside article card at the bottom

Reply:
Cool.

Reply:
Worked on this